gapic/generator/generator.py

```diff
@@ -297,6 +296,14 @@ def _render_template(
             template_name, api_schema=api_schema, opts=opts))
         return answer
 
+    def _is_desired_transport(self, template_name: str, opts: Options) -> bool:
+        """Returns true if template name contains a desired transport"""
+        desired_transports = ['__init__', 'base'] + opts.transport
+        for transport in desired_transports:
+            if transport in template_name:
+                return True
+        return False
```

Comment:
This is such a common pattern that python has a builtin function for determining it.
```python
return any(transport in template_name for transport in desired_transports)
```